src/ProtocolV3TestBase.sol

```diff
@@ -312,12 +312,13 @@ contract ProtocolV3TestBase is CommonTestBase {
     ReserveConfig memory borrowConfig,
     uint256 collateralAmount
   ) internal view returns (uint256) {
+    // Intentionally introducing a slight rounding error to not trigger the LTV edge case failure condition
```

Comment:
Wouldn't it be better to subtract 1 from the borrowable amount or add 1 to the collateral added? It would look much simpler IMO instead of introducing a variable rounding error there

Reply:
In theory this is the exact boundary value.
Even though we might have introduced the a slight rounding error (we actually did in only 1 case) I'd keep this instead of explicitly going below the real boundary in every single case.
Also with this solution, we could have kept the epsilon value without adding +1.